### PR TITLE
Adding  redshift to the configuration dictionary

### PR DESCRIPTION
**Pull request by bazkiaei**
The goal of this PR is to add the redshift of the galaxy at the distance chosen by the user to the configuration dictionary. By this, the redshift is computed once and is contained in the configuration dictionary #71 .

### CODE REVIEW

File: mocks/mocks.py [hunk 1]
```diff
@@ -137,10 +138,15 @@ def parse_config(config_file='config_example.yaml',
                                 config['m_nu'])
     config['m_nu'] = ensure_unit(config['m_nu'],
                                  u.eV)
-    # Redshift
+    # Redshift, from the simulation.
     config['redshift'] = kwargs.get('redshift',
                                     config['redshift'])
 
+    # This is the redshift of the galaxy in the selected position by the user.
+    cosmo = mocks.create_cosmology(config)
```
Comment (bazkiaei) [inline mocks/mocks.py]: This will be modified when PR #85 is approved and merged.

**Comment by bazkiaei** [comment]
@AstroHuntsman/spiderlings 
Ready for review.

File: mocks/mocks.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
+from mocks import mocks
```
Comment (wtgee) [inline mocks/mocks.py]: Isn't this mocks.mocks?

Comment (bazkiaei) [reply mocks/mocks.py]: Wow, such a dummy mistake I made! 🌝

**Comment by bazkiaei** [comment]
With respect to the advice that I received, redshift should not be added to the configuration, so, I am closing this PR.

**Pull request closed without merging**